### ISSUE TITLE
Redshift crash on Arch Linux: UnicodeDecodeError

### ISSUE THREAD

**Issue by siddharthist** [jonls/redshift#271]
See title. The following error occurs on any `redshift-gtk` command, but not on `redshift` commands. It might be a Python version bug? Arch uses Python3 by default.  Let me know if I can provide more information! 
```
$ redshift -V
redshift 1.10
```
```
$ sudo pacman -Q redshift
redshift 1.10-2
```
```
$ python --version           
Python 3.5.0
```
```
$ redshift-gtk
Traceback (most recent call last):
  File "/usr/lib/python3.5/tokenize.py", line 392, in find_cookie
    line_string = line.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 40: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/redshift-gtk", line 26, in <module>
    from redshift_gtk.statusicon import run
  File "/usr/lib/python3.5/site-packages/redshift_gtk/statusicon.py", line 32, in <module>
    from gi.repository import Gtk, GLib, GObject
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 634, in _load_backward_compatible
  File "/usr/lib/python3.5/site-packages/gi/importer.py", line 115, in load_module
    dynamic_module = load_overrides(introspection_module)
  File "/usr/lib/python3.5/contextlib.py", line 66, in __exit__
    next(self.gen)
  File "/usr/lib/python3.5/site-packages/gi/importer.py", line 78, in _check_require_version
    PyGIWarning, stacklevel=stacklevel)
  File "/usr/lib/python3.5/warnings.py", line 18, in showwarning
    file.write(formatwarning(message, category, filename, lineno, line))
  File "/usr/lib/python3.5/warnings.py", line 26, in formatwarning
    line = linecache.getline(filename, lineno) if line is None else line
  File "/usr/lib/python3.5/linecache.py", line 16, in getline
    lines = getlines(filename, module_globals)
  File "/usr/lib/python3.5/linecache.py", line 47, in getlines
    return updatecache(filename, module_globals)
  File "/usr/lib/python3.5/linecache.py", line 136, in updatecache
    with tokenize.open(fullname) as fp:
  File "/usr/lib/python3.5/tokenize.py", line 456, in open
    encoding, lines = detect_encoding(buffer.readline)
  File "/usr/lib/python3.5/tokenize.py", line 433, in detect_encoding
    encoding = find_cookie(first)
  File "/usr/lib/python3.5/tokenize.py", line 397, in find_cookie
    raise SyntaxError(msg)
SyntaxError: invalid or missing encoding declaration for '/usr/bin/sys'
```

**Comment by andersk** [comment]
This happens whenever you have a file named `sys` in the same directory as `redshift-gtk`. Some relevant upstream bugs are [PyGObject #757184](https://bugzilla.gnome.org/show_bug.cgi?id=757184) and [Python #25493](https://bugs.python.org/issue25493).

However, that doesn’t absolve `redshift-gtk` of all responsibility. That bug is triggered because PyGObject is trying (and failing) to display these warnings:

```
sys:1: PyGIWarning: Gtk was imported without specifying a version first. Use
gi.require_version('Gtk', '3.0') before import to ensure that the right version
gets loaded.
sys:1: PyGIWarning: AppIndicator3 was imported without specifying a version
first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure
that the right version gets loaded.
```

`redshift-gtk` should resolve these warnings as suggested, both because of the upstream display bug and because warnings are bad even when correctly displayed.

**Comment by andersk** [comment]
This was already resolved by 442fb9bd5df72e6eb89c34352882ac80f9c9660b and 56cef8d9cc2474e6e7382f9e702d35ab7f6355a8 (both following v1.10).

**Comment by jonls** [comment]
Thanks @andersk! This should be fixed in the next release then.
